src/libraries/Microsoft.PowerFx.Connectors/Tabular/Services/CdpTable.cs

```diff
@@ -146,9 +146,13 @@ private async Task<string> Query(IServiceProvider serviceProvider, DelegationPar
         }
 
         private IReadOnlyCollection<DValue<RecordValue>> GetResult(string text)
-        {
-            // $$$ Is this always this type?
-            RecordValue rv = FormulaValueJSON.FromJson(text, RecordType.Empty().Add("value", TableType)) as RecordValue;
+        {            
+            if (FormulaValueJSON.FromJson(text, RecordType.Empty().Add("value", TableType)) is not RecordValue rv)
+            {
+                // FromJson can return an ErrorValue and in that case let's return an empty result
```

Comment:
> empty result [](http://example.com/codeflow?start=85&length=13)

My concern is dropping the error. 

Should we use DValue.Of(ErrorValue) and propagate the error through?

Reply:
Let's propagate the error properly